### ISSUE TITLE
Users need to provide an address that's different from their verified address so that they can receive their permit/pass

### ISSUE THREAD

**Issue by ghost** [alphagov/verify-local-patterns#321]
I don't know if we'll be able to modify addresses at this point if the application has been submitted and all the internal messaging has been sent off. It would be easier to not include this functionality, and instead give a contact detail in case someone wants to change it.

**Comment by joanneSouthern** [comment]
Agree, i wouldnt want to verify an address and then send information out to another address

**Comment by irinapencheva** [comment]
@samuelculley - could we chat what we mean about that

**Comment by suryanaga** [comment]
@samuelculley I'm not sure what you mean by "the application has been submitted and all the internal messaging has been sent off". As this a prototype, we haven't specified yet what the back end will do, but when we do, we can say when the data gets sent.

**Comment by suryanaga** [comment]
Also, just to say this is important functionality as users may need to get their pass sent to their work or another address they can more easily retrieve it from.

**Comment by irinapencheva** [comment]
@sanjaypoyzer - I think this would depend on the individual council service risk appetite and also needs to be confirmed by Julian that it's OK from a risk & assurance perspective. If a user can change the address of their pass delivery once the application is complete, there is no physical assurance that the pass will actually be delivered to an address in the council (i.e. potential fraud vector). Other council services don't allow such functionality because this is one way to assure the address is real and w/in the council boundaries.

**Comment by suryanaga** [comment]
@irinapencheva Well it's quite easy for the service to be checking that the new address entered at this point is also within the council boundaries. If that's the only reason a council wouldn't want to accept a new address at this point, I don't think that's a real issue.

**Comment by irinapencheva** [comment]
@sanjaypoyzer I'll check with councils what they accept and will also confirm with Julian

**Comment by irinapencheva** [comment]
Also then the wording on the below screen would need amend, so let's confirm first what's should be logic from Risk & Assurance point
![screen shot 2017-04-21 at 13 12 03](https://cloud.githubusercontent.com/assets/24913176/25276908/533095ca-2694-11e7-8b5f-089c495c6234.png)

**Comment by irinapencheva** [comment]
@sanjaypoyzer - just confirmed with Julian and this functionality should not be allowed at the end because it enables a huge risk vector. This applies for both concessionary travel and parking permits

**Comment by suryanaga** [comment]
@irinapencheva How is this a risk? I'm going to need more information than that as it's clearly a valid user need that we should meet.

**Comment by irinapencheva** [comment]
@sanjaypoyzer - if a user can change their address after the application has been effectively processed, then there is no assurance for the service that they actually live at the address - i.e. I can apply for a permit to an address that fits the councils's eligibility criteria and then change it to anything and ultimately allow quite a lot of fraud. 
@EUzkuraityte - is there anything you want to add to the above?

Happy to discuss as part of the daily prioritisation stand-up and maybe chat to Julian on why that's the case.

**Comment by EUzkuraityte** [comment]
@irinapencheva @sanjaypoyzer agree with Irina's point. Julian and Alasdair also requested to review the prototypes so might be more issues they pick up on.

**Comment by suryanaga** [comment]
@irinapencheva But we still know who the user is. What is actually the fraud scenario we are protecting against here? I still haven't heard a convincing scenario.

**Comment by irinapencheva** [comment]
@sanjaypoyzer - yes, you might know who I am but then I can: 1. get verified with an address outside of the council 2. change to an address w/in council boundary 3. get confirmation that the pass is processed 4. change my address again at the confirmation page to my 'office address'
(and it really doesn't matter what kind of legal checkbox I've clicked)

Then I can order bus passes cards from +20 councils, get them delivered to my 'office address' and sell them online.

Check with Julian if you're still not convinced but ultimately they've said we can't have this option

**Comment by suryanaga** [comment]
@irinapencheva Suggest changing the title of this to "Users need to provide an address that's different from their verified address so that they can receive their permit/pass "

**Comment by irinapencheva** [comment]
@sanjaypoyzer - again I'm not sure we saw this as a user need from any previous user research at this stage of the transaction (i.e the end of the transaction). Let's discuss in person

I will discuss this with Julian in more detail and will share with you everything we capture around whether it's is a fraud risks

**Comment by suryanaga** [comment]
@irinapencheva Whether or not we've seen it in research, it's the assumed underlying user need that this issue is about

**Comment by irinapencheva** [comment]
OK, will rephrase if that makes more sense although that's not an observed UR.
For the record the issue is currently:

'Users are allowed to provide a different address at the end of the application but this might present a significant fraud risk'

**Comment by irinapencheva** [comment]
@sanjaypoyzer - I have just spoken to Julian and this is a very clear fraud vector because I can apply based on other people's details and if the letter goes to another address, the genuine user who the application was on behalf of wouldn't even get notified/ know about it. 

For parking the fraud vector is somewhat less strong, because the value is limited but it's still not great and needs to be confirmed by services if they're willing to accept this. He has requested that we remove this functionality. 

In addition, this is not a user need we've seen in user research and as per the comment from Northumberland address, services don't want to do this.

**Comment by suryanaga** [comment]
@irinapencheva Can you set up a meeting with me and Julian to talk about this? I don't understand the fraud scenario from your description.
